docs/development/git_workflow.rst

```diff
@@ -536,7 +536,24 @@ implement your work.
 After committing your changes you push your change to your repository
 `git push origin helping-with-PR116`.
 
- 
+
+Documentation Guidelines
+========================
+
+Building local docs is done via compiling docs locally. In order to do this, in the terminal type::
```

Comment:
Redundant wording. Perhaps "These instructions cover how to compile documentation locally."

Reply:
Done!